docs/source/blogs/tech_blog/blog10_ADP_Balance_Strategy.md

```
@@ -0,0 +1,209 @@
+- `sol_tps` represents the theoretical upper-bound throughput.
+
+
+### Scheduling Strategies for Load Balancing
+In any given iteration, the Attention module can face significant differences in the number of tokens among ranks. The total execution time for this part is bound by the rank with the longest processing time.
+
+#### Baseline
+One approach is to sort the requests in the request queue by `num_tokens` and then dispatch them to different ranks in a round-robin fashion, as shown in the figure below. This method balances the total number of tokens across ranks from a global perspective, and can only reduce the disparity in token numbers between different contexts when all ranks are processing context requests.
+
+<div align="center">
+<figure>
+  <img src="./../media/tech_blog10_baseline_round_robin_strategy.png">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)